app/src/main/java/net/gsantner/markor/activity/MainActivity.java

```diff
@@ -101,10 +102,14 @@ public void onPageSelected(int position) {
         optShowRate();
 
         // Setup viewpager
-        _viewPager.setAdapter(new SectionsPagerAdapter(getSupportFragmentManager()));
-        _viewPager.setOffscreenPageLimit(4);
+        _sectionsAdapter = new SectionsPagerAdapter(getSupportFragmentManager());
```

Comment:
When main activity is loaded we do not load not shown tabs immediately. 

Instead we load them lazily. 

After tabs are loaded, they are cached by the adapter